src/components/menuBar/menu-bar.spec.js

```diff
@@ -279,6 +279,108 @@ describe('material.components.menuBar', function() {
           return el;
         }
       });
+
+      describe('nested menus - close menu with escape key', function() {
+        var menuBar, menus, subMenuOpen, ctrl;
+
+        it('opens consecutive nested menus', inject(function($mdConstant) {
```

Comment:
This name doesn't seem representative of what's being tested. It seems like the `describe` text is what should be here?

Also it should probably be `closes only the top level menu with the escape key`.